airflow/providers/amazon/aws/hooks/sns.py

```diff
@@ -52,7 +53,7 @@ class AwsSnsHook(AwsBaseHook):
     def __init__(self, *args, **kwargs):
         super().__init__(client_type='sns', *args, **kwargs)
 
-    def publish_to_target(self, target_arn, message, subject=None, message_attributes=None):
+    def publish_to_target(self, target_arn, message, subject=None, message_attributes: Optional[dict] = None):
```

Comment:
Can you add TypeHints to `target_arn`, `message` and `subject` too please

Reply:
Sure, I might have missed it. I will add those!